nailgun/entities.py

```diff
@@ -5467,6 +5467,114 @@ def sync(self, synchronous=True, **kwargs):
         return _handle_response(response, self._server_config, synchronous)
 
 
+class ProductsBulkActions(Entity):
+    """A representation of a Products bulk actions entity."""
+
+    def __init__(self, server_config=None, **kwargs):
+        self._meta = {
+            'api_path': '/katello/api/products/bulk',
+            'server_modes': ('sat', 'sam'),
```

Comment:
no need of adding server_modes as we are not using it now.

Reply:
updated PR.